packages/manager/modules/hub/src/components/products/products.html

```diff
@@ -7,11 +7,12 @@
             heading="{{:: 'manager_hub_products_' + product | translate }}"
             badge-text="{{ data.count }}"
             class="oui-list"
+            data-on-click="$ctrl.onProductSelect({ productType: product })"
         >
             <ul class="oui-list__items">
                 <li
                     class="oui-list__item"
-                    data-ng-repeat="service in data.data track by service.serviceId"
+                    data-ng-repeat="service in data.data  | limitTo: 4 track by service.serviceId"
```

Comment:
This limit, could be stored in constant to be shared with the one used in the routing

Reply:
Not the same constant used, they do not impact the same component and don"t have the same purpose but I agree it could be more explicit